src/SpacePlanning/Generate/Packers/CuboidPacker.cs

```diff
@@ -65,8 +67,8 @@ private void PackItems(List<Item> items)
             var packingResult = containerPackingResult.FirstOrDefault().AlgorithmPackingResults.FirstOrDefault();
             if (packingResult == null) throw new InvalidOperationException(PackingFailed);
 
-            // record results in this packer instance
-            this.PackedItems = CuboidsFromItems(packingResult.PackedItems);
+            // record results in this packer instance            
+            this.PackedItems = TransformPackedCuboids(this.containerCuboid, CuboidsFromItems(packingResult.PackedItems));
```

Comment:
we should separate this out on 2 lines as it's 2 distinct operations : transformation (`CuboidsFromItems`) and then moving (`TransformPackedCuboids`)

Reply:
resolved in #134